### ISSUE TITLE
Scheduled messages. Info and Relationships

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1455]
## Description of issue

Scheduled messages are presented in a way that communicates the relationship between elements visually, but this relationship is not defined programmatically. This relationship can be communicated by using table markup with items in the first column as row headers, or presented as a definition list (although support for definition lists is ok, but not as good as for tables, see Brief note on description list support). 

<img width="716" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUg0QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a333c3ae844759bcbe7a1013d70581c77709e4a1/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202024-03-20%20a%CC%80%2015.00.19.png" alt="Capture d’écran, le 2024-03-20 à 15.00.19.png" />

## Potential fix

- Use a table

## Resources

- [Description-list support](https://adrianroselli.com/2022/12/brief-note-on-description-list-support.html)

**Comment by amazingphilippe** [comment]
while working on this page, we should address the lack of feedback when cancelling a job. As described in #1470

- [ ] Flash a message on the dashboard after cancelling a job, to confirm that "job name" was indeed cancelled. This should be a status banner, and get focus on page load.

**Comment by andrewleith** [comment]
- @whabanks to finish this off today

**Comment by yaelberger-commits** [comment]
tests were broken, Will fixed those, ready for review

**Comment by YedidaZalik** [comment]
Finished comments on PR and needs to do a merge conflict 
Phil will review again

**Comment by foudamo** [comment]
Waiting for Andrew to review
